integrationTests/multiShard/block/executingRewardMiniblocks_test.go

```diff
@@ -0,0 +1,223 @@
+package block
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExecuteBlocksWithOnlyRewards(t *testing.T) {
```

Comment:
WithOnlyRewards doesn't sound too good. Might rename this to "TestExecuteBlocksWithTransactionsAndCheckRewards"
Might have another test for blocks without transactions.

Reply:
done